src/rpdk/core/invoke.py

```diff
@@ -23,7 +23,13 @@ def invoke(args):
     project.load()
 
     client = ResourceClient(
-        args.function_name, args.endpoint, args.region, project.schema, {}
+        args.function_name,
+        args.endpoint,
+        args.region,
+        args.partition,
+        args.account_id,
+        project.schema,
+        {},
```

Comment:
what's that for? why empty `dict` is injected?

Reply:
empty dict is for overrides. This is not a change which was made in this PR.

Reply:
injecting an empty dict does not seem right.. should we set a default to `None` in the target function?

Reply:
I am relunctant to make an unrelated code change in this CR.